core/math/random_pcg.cpp

```diff
@@ -51,16 +51,7 @@ float RandomPCG::random(float p_from, float p_to) {
 }
 
 int RandomPCG::random(int p_from, int p_to) {
-	int range;
-	int min;
-	if (p_to > p_from) {
-		range = p_to - p_from + 1;
-		min = p_from;
-	} else if (p_to < p_from) {
-		range = p_from - p_to + 1;
-		min = p_to;
-	} else { // from == to
+	if (p_from == p_to)
```

Comment:
Use curly braces for that expression - that's a rule which accepted for the entire 
engine.